### ISSUE TITLE
Cannot open token authentication

### ISSUE THREAD

**Issue by Steve0x2a** [apache/pulsar#14591]
**Describe the bug**
Cannot open token authentication.

**To Reproduce**
1. download pulsar 2.9.1 in release
2. edit `conf/standalone.conf`
```conf
# Enable authentication
authenticationEnabled=true

# Authentication provider name list, which is comma separated list of class names
authenticationProviders=org.apache.pulsar.broker.authentication.AuthenticationProviderToken

# Enforce authorization
authorizationEnabled=true

# Authorization provider fully qualified class-name
authorizationProvider=org.apache.pulsar.broker.authorization.PulsarAuthorizationProvider

tokenSecretKey=/path/to/my-secret.key
```
No other configuration changes
3. run `bin/pulsar standalone`

**Expected behavior**
pulsar runs normally and opens token authentication

**Screenshots**
![YI87WG7C9(B6 5Y`C1MV}W](https://user-images.githubusercontent.com/17063270/157150876-f1b8c0d2-aaa6-42ac-bf28-4993d0fd4ae8.jpg)

**Desktop (please complete the following information):**
 - OS: Ubuntu

**Comment by nodece** [comment]
Did you follow the https://pulsar.apache.org/docs/en/security-jwt/#enable-token-authentication?

**Comment by Steve0x2a** [comment]
@nodece Yes, I generated the key according to the documentation, and modified the configuration file, but when I run pulsar I get an error.

**Comment by nodece** [comment]
Please provide the full log.

**Comment by Steve0x2a** [comment]
[log.txt](https://github.com/apache/pulsar/files/8249868/log.txt)
@nodece

**Comment by nodece** [comment]
Workaround:
1. You need to update the auth in `client.conf`
2. Start standalone by `pulsar standalone -nfw`
